Microsoft.Sbom.sln

```diff
@@ -30,6 +30,16 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Sbom.Api.Tests",
 EndProject
```

Comment:
Need to add the new parameter here as well:
https://github.com/microsoft/sbom-tool/blob/e1b2e30d5cde6b7d37e9c287f1528719ca577f1c/README.md?plain=1#L57